src/_ctf-private/ctf-analytics/CtfSegmentAnalytics.tsx

```diff
@@ -26,12 +26,11 @@ export const CtfSegmentAnalytics = () => {
     if (initialized && analytics) {
       analytics.page({
         templateId: 'marketing',
-        xRayActive: Boolean(xray),
         previewActive: Boolean(preview),
         guestSpaceActive: Boolean(guestSpaceActive),
       });
     }
-  }, [guestSpaceActive, initialized, preview, xray]);
+  }, [guestSpaceActive, initialized, preview]);
```

Comment:
Once we remove xray, we should notify Rosi and update the Segment tracking plan for starter-templates accordingly

Reply:
we can only adjust the segment plan once we've adjusted this for every starter-template?

Reply:
That's a good shout - hadn't considered that. Alright for now we should be okay to ignore it; I think Segment still sends the end

Reply:
Ok I add it to the follow up tickets for the remaining starter templates

Reply:
For reference, this is the [document](https://contentful.atlassian.net/wiki/spaces/PROD/pages/4138500311/Starter+Templates+-+Segment+tracking#Tracking-plan) that needs to be updated once the event is removed from the Tracking plan